src/qmckl_ao.org

```diff
@@ -1,3 +1,7 @@
+#+begin_src elisp :results none :noexport:
+(org-babel-lob-ingest "./orglib")
+#+end_src
+
```

Comment:
OK. So maybe it did not work when we tried it for other reasons. Good you solved it!

Reply:
Yes! Not sure why it didn't work before. Could we rename the orglib file to something with a .org extension ? I didn't test this.